extensions/wikia/PhalanxII/services/PhalanxService.class.php

```diff
@@ -313,9 +313,15 @@ private function sendToPhalanxDaemon( $action, $parameters ) {
 	}
 
 	private function getPhalanxUrl( $action ) {
-		global $wgConsulUrl, $wgConsulServiceTag;
-		
-		$baseurl = ( new ConsulUrlProvider( $wgConsulUrl, $wgConsulServiceTag ) )->getUrl( 'phalanx' );
+		global $wgConsulUrl, $wgConsulServiceTag, $wgPhalanxServiceUrl;
+
+		if ( !empty( $wgPhalanxServiceUrl ) ) {
+			$baseurl = $wgPhalanxServiceUrl;  // e.g. "localhost:4666"
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)